vlib/v/fmt/tests/type_decl_with_anon_struct_keep.vv

```diff
@@ -0,0 +1,15 @@
+pub type Unit1 = struct {}
+
+pub type Unit2 = struct {
+		name string
+		age  int
+	}
```

Comment:
That indented `}` looks a bit odd to me.

Reply:
Ok, I'll fix it.